### PR TITLE
feature(ui-ux): add chain related stats display

### PR DESCRIPTION
**Pull request by veralygit**
#### What this PR does / why we need it:
This PR pushes chain related stats display features 

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
preview : https://deploy-preview-7--defimetascan.netlify.app/

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: src/components/commons/ChainStatsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import BigNumber from "bignumber.js";
+
+interface StatsCardProps {
+  cardTitle: string;
+  cardBody: string;
+  cardFooter: string;
+  classNames?: string;
+}
+
+export function useUnitSuffix(
```
Comment (kyleleow) [inline src/components/commons/ChainStatsCard.tsx]: let's refactor this to its own file

File: src/components/commons/ChainStatsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import clsx from "clsx";
+import BigNumber from "bignumber.js";
+
+interface StatsCardProps {
+  cardTitle: string;
```
Comment (kyleleow) [inline src/components/commons/ChainStatsCard.tsx]: let's remove `card..` for the three props naming here, if we have two title maybe, then adding a differentiator will help, but right now this is not the case

File: src/layouts/components/GroupChainStatsCard.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import ChainStatsCard from "@components/commons/ChainStatsCard";
+import Container from "@components/commons/Container";
+
+function GroupChainStatsCard() {
+  const groupStatsCardContent = [
+    {
+      cardTitle: "24h total transactions",
+      cardBody: "32324000000",
+      cardFooter: "+12.34% last 30 days",
+      classNames: "bg-gradient-to-br from-[#42F9C2] to-[#082FD4]",
```
Comment (kyleleow) [inline src/layouts/components/GroupChainStatsCard.tsx]: we have the gradient defined in `global.css`, check `brand-gradient-1` and `2`

File: src/layouts/components/GroupChainStatsCard.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    {
+      cardTitle: "24h total volume liquidity",
+      cardBody: "3232400000",
+      cardFooter: "+2.34% last 24H",
```
Comment (kyleleow) [inline src/layouts/components/GroupChainStatsCard.tsx:16]: latest figma removed `.. last 24H`

File: src/layouts/components/GroupChainStatsCard.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import ChainStatsCard from "@components/commons/ChainStatsCard";
+import Container from "@components/commons/Container";
+
+function GroupChainStatsCard() {
+  const groupStatsCardContent = [
+    {
+      cardTitle: "24h total transactions",
+      cardBody: "32324000000",
+      cardFooter: "+12.34% last 30 days",
```
Comment (kyleleow) [inline src/layouts/components/GroupChainStatsCard.tsx:10]: latest figma removed `... last 30 days`

File: src/components/commons/ChainStatsCard.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  return (
+    <div
+      className="rounded-2xl w-full h-full p-[0.3px] shadow-[0_3px_6px_rgba(0,0,0,0.2)]"
+      style={{
+        background:
+          "linear-gradient(152.58deg,rgba(255, 255, 255, 0.4),rgba(255, 255, 255, 0))",
+      }}
+    >
+      <div className="bg-black-900 rounded-2xl rounded-2xl w-full h-full">
+        <div className="rounded-2xl flex flex-col justify-center pl-10 black-gradient-1 py-8">
```
Comment (kyleleow) [inline src/components/commons/ChainStatsCard.tsx]: why left padding only? let's apply on both side, to prevent ugly UI in the future if there is long text

Comment (kyleleow) [reply src/components/commons/ChainStatsCard.tsx]: add diff padding for tablet and mobile view

File: src/components/commons/ChainStatsCard.tsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+  return (
+    <div
+      className="rounded-2xl w-full h-full p-[0.3px] shadow-[0_3px_6px_rgba(0,0,0,0.2)]"
+      style={{
+        background:
+          "linear-gradient(152.58deg,rgba(255, 255, 255, 0.4),rgba(255, 255, 255, 0))",
+      }}
+    >
+      <div className="bg-black-900 rounded-2xl rounded-2xl w-full h-full">
```
Comment (kyleleow) [inline src/components/commons/ChainStatsCard.tsx]:
```suggestion
      <div className="bg-black-900 rounded-2xl w-full h-full">
```

File: src/components/commons/ChainStatsCard.tsx [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+        <div className="rounded-2xl flex flex-col justify-center pl-10 black-gradient-1 py-8">
+          <div className="text-white-700 font-medium text-xs tracking-wider uppercase ">
```
Comment (kyleleow) [inline src/components/commons/ChainStatsCard.tsx]:
```suggestion
          <div className="text-white-700 font-medium text-xs tracking-wider uppercase">
```

File: src/components/commons/ChainStatsCard.tsx [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+            {cardTitle}
+          </div>
+          <div className="text-white-50 text-2xl font-light tracking-wider py-2">
+            {isSuffixRequired ? valueToUnitSuffix : cardBody}
+          </div>
+          <div
+            className={clsx(
+              " text-transparent bg-clip-text font-medium text-xs tracking-wider uppercase max-w-[146px]",
```
Comment (kyleleow) [inline src/components/commons/ChainStatsCard.tsx]:
```suggestion
              "text-transparent bg-clip-text font-medium text-xs tracking-wider uppercase max-w-[146px]",
```

File: src/layouts/components/GroupChainStatsCard.tsx [hunk 10]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import ChainStatsCard from "@components/commons/ChainStatsCard";
+import Container from "@components/commons/Container";
+
+function GroupChainStatsCard() {
```
Comment (kyleleow) [inline src/layouts/components/GroupChainStatsCard.tsx]: any reason to have this component in `.../layouts/components`?

Comment (veralygit) [reply src/layouts/components/GroupChainStatsCard.tsx]: placed in the wrong components folder have updated to `.../src/components`

File: src/components/commons/ChainStatsCard.tsx (hunk 8 above)
Comment (kyleleow) [reply src/components/commons/ChainStatsCard.tsx]: not fixed yet

**Review by kyleleow — APPROVED** [review]

**Pull request merged**